codebuild_validation.yml

```diff
@@ -21,6 +21,7 @@ phases:
     runtime-versions:
       nodejs: latest
     commands:
+      - apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb # https://on.cypress.io/required-dependencies
```

Comment:
Fra listen med avhengigheter her: https://on.cypress.io/required-dependencies

Ser det blir gjort noe lignende i e2e-prosjektet også: https://github.com/BIBSYSDEV/NVA-end-to-end-testing/blob/develop/buildspec.yml#L10

Men får installasjonsfeil:
![bilde](https://user-images.githubusercontent.com/6313468/186904691-33023d7e-2d2d-464e-ab8f-2a772d5a164c.png)